### PR TITLE
AFBA housekeeping

### PR DESCRIPTION
**Pull request by lostella**
@pylat do you approve these changes? They are all minor. I shouldn't have broken anything though. I would also ask you to complete the docstring at the bottom, with the description of the parameters: these are important because allow you to do `?AFBA` or `?ProximalAlgorithms.AFBA` and get the help page in the terminal. You should be able to contribute to this PR, if necessary, by pushing commits to the `afba-fixes` branch.

### CODE REVIEW

**Comment by pylat** [comment]
Great! Yes it looks much better. Just a small comment about the docstrings.

**Comment by nantonel** [comment]
maybe the table and references in `README.md` could be also updated? :smile:

**Comment by lostella** [comment]
@pylat I added dual vector to the returned values + a test that solves a random LP that I was playing with yesterday. I thought "why not".

I also added shortcuts to Chambolle-Pock and Vu-Condat: these are very famous algorithms, I think it is important to directly reference to them. Can you check that these are indeed correct?

File: src/algorithms/AsymmetricForwardBackwardAdjoint.jl [hunk 1]
```diff
@@ -1,19 +1,5 @@
...
-# The iterator implements Algorithm 3 of [1] with constant stepsize (α_n=λ) for several prominant special cases:
-# 1) θ = 2 	 	==>   Corresponds to the Vu-Condat Algorithm [2,3].
-# 2) θ = 1, μ=1
-# 3) θ = 0, μ=1
-# 4) θ ∈ [0,∞), μ=0 
-#
-# See [1, Figure 1] for other special cases and relation to other algorithms.   
+# Primal-dual algorithms based on Asymmetric Forward-Backward-Adjoint
 #
 # [1] Latafat, Patrinos. "Asymmetric forward–backward–adjoint splitting for solving monotone inclusions involving three operators"  Computational Optimization and Applications, pages 1–37, 2017.
```
[the remark this answers is not in the document]
Comment (lostella) [reply src/algorithms/AsymmetricForwardBackwardAdjoint.jl]: I guess references can stay in both places. I put them on top next to the “title” of the file just for convenience

**Pull request merged**